src/components/carousel/carousel-styles.scss

```diff
@@ -0,0 +1,63 @@
+$bottom-padding: 16px;
+
+.carouselWrapper {
+  font-family: 'Lato', sans-serif;
+  display: flex !important;
```

Comment:
Do we need this !important s? It seems to work well without them. Also the :global are annoying but I suppose there is no other way to access that classes

Reply:
You're right those `!importants` are not needed anymore, thanks!!
I'm leaving the `:global .cwCarouselPaging` one to avoid library inline styles precedence.